corehq/toggles/__init__.py

```diff
@@ -2600,14 +2600,13 @@ def _handle_attendance_tracking_role(domain, is_enabled):
     save_fn=_handle_attendance_tracking_role,
 )
 
-GEOSPATIAL = StaticToggle(
-    'geospatial',
-    'Allows access to GIS functionality',
+MICROPLANNING = StaticToggle(
+    'microplanning',
```

Comment:
Hey @zandre-eng 

Just a suggestion that you can retain the same feature flag if you retain the same slug which I think isn't bad unless the switch over is easy.

Reply:
Hey @mkangia,

The switchover is fortunately fairly easy. On production there are mostly just test domains using this feature flag and they will simply be manually switched over to the new feature flag toggle once it's live. I think it's a good opportunity to switch it over now while this feature isn't being widely used yet.